toqito/nonlocal_games/tests/test_nonlocal_game.py

```diff
@@ -251,3 +251,95 @@ def test_unbalanced_nonlocal_game(self):
 
         # Expected result: We do not care for the value, just triggering the block
         self.assertIsNotNone(res)
+        
+    def test_chsh_classical_value_fast(self):
```

Comment:
Presumably the fast method is now the default, so we can likely rename this test. Maybe we can refer to this as `test_chsh_bcs_game` or something similar?